### ISSUE TITLE
Feature request: support Bytes fields

### ISSUE THREAD

**Issue by derekwheel** [47ng/prisma-field-encryption#26]
Hello!
It looks like only string fields are currently supported, but Prisma also supports a `Bytes` type: https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#bytes
It seems natural that one would want to encrypt fields of this kind as well, are there any plans to support it?

**Comment by franky47** [comment]
That would seem like a good improvement indeed.

Would you like to open a PR?

**Comment by huksley** [comment]
Also interest in this. I keep file contents in Bytes type field and it sometimes can contain sensitive data (SSH private key for example)

**Comment by franky47** [comment]
Yeah that sounds like something you'd want to encrypt. Would you feel up to opening a PR?

**Comment by huksley** [comment]
Unfortunately not right now. I don't feel confident hacking Prisma - my previous attempts at writing client extensions failed.
